app/javascript/packs/components/rdv-motif.js

```diff
@@ -0,0 +1,28 @@
+class RdvMotif {
+
+  toggleMotif(motif) {
+    let rdvAtHome = $("input[type=checkbox][name='motif[at_home]']").is(":checked")
```

Comment:
I guess this should be a `const` as it's not reassigned afterwise. 
which makes me realize we don't have a JS linter in place, but that's another discussion